packages/wrapped-keys/src/lib/constants.ts

```diff
@@ -1,31 +1,2 @@
-const CHAIN_ETHEREUM = 'ethereum';
-const LIT_PREFIX = 'lit_';
-export const NETWORK_EVM = 'evm';
-export const NETWORK_SOLANA = 'solana';
```

Comment:
Why has this been removed?

Reply:
Same reason as https://github.com/LIT-Protocol/js-sdk/pull/513#discussion_r1658783156; consumers of the SDK get IDE auto-complete and TS compiler checking based on the string union type defined here: https://github.com/LIT-Protocol/js-sdk/blob/c3dafd0a4eb4ccf7bb93848686bbd04a01882fd0/packages/wrapped-keys/src/lib/types.ts#L10